js/components/ConnectionBar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { StyleSheet, View, Text } from 'react-native'
+import { connect } from 'react-redux'
+
+import I18n from '../i18n'
+
+class ConnectionBar extends React.Component {
```

Comment:
add a short comment describing the purpose of this component

Reply:
also, extend [PureComponent](https://60devs.com/pure-component-in-react.html) instead of `Component` when possible (almost always)